scripts/dashboard/medperf_dashboard/get_data.py

```diff
@@ -0,0 +1,126 @@
+import os
+import pandas as pd
+import datetime
+
+from medperf.entities.dataset import Dataset
+from medperf import config
+
+from .utils import get_institution_from_email, get_reports_path, stage_id2name
+
+from typer import Typer, Option, run
+
+app = Typer()
+
+
+def get_dsets(mlcube_id):
+    dsets = Dataset.all(filters={"mlcube": mlcube_id})
+    dsets = [dset.todict() for dset in dsets]
+    for dset in dsets:
+        user_id = dset["owner"]
+        dset["user"] = config.comms.get_user(user_id)
```

Comment:
how many API calls are we expecting here? maybe we can have an endpoint that returns all of this

Reply:
I expect here a large number of unnecessary API calls (this includes multiple datasets for the same user, so we are talking more than 30 maybe?)
let's have an issue to replace this code later. in the FL PR we will have an endpoint that lists experiment datasets and their owners information so we can reuse the logic defined there to make this better.